### PR TITLE
Bug with execution an action repeatedly fixed (close #663)

### LINKED ISSUE #663: Driver executes an action repeatedly if it leads to the page reloading on iOS
Webkit based browsers on ios don't raise `beforeunload` event. They raise just `pagehide` event and it works like the same way as `unload` event. As a result some our scripts (that should be executed on `beforeunload` event) are not executed because of page unloading.

### PR DESCRIPTION
**Pull request by helen-dikareva**
waiting for passing tests on sauce labs

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
Can you give me some context, because I have no idea what this fix does?

**Comment by AlexanderMoskovkin** [comment]
@inikulin,
The problem:
1) Driver sends a status via hammerhead transport
2) The request is aborted (because the page is unloading)
3) Hammerhead store the message in the context storage (to resend it after page reload) and [call the callback](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/transport.js#L121)
4) Since callback is called Driver think that the message is delivered but it's wrong.

In other words driver has his own ensure message delivering mechanizm. So we don't need such logic in transport. So, we should set the `disableResending` flag to `true` when send a status.

**Comment by inikulin** [comment]
@AlexanderMoskovkin Thank you for the explanation. lgtm then.
